protocol/revapi.json

```diff
@@ -84,6 +84,32 @@
           "code": "java.method.removed",
           "old": "method java.util.Set<java.lang.Long> io.camunda.zeebe.protocol.record.value.ProcessInstanceModificationRecordValue::getActivatedElementInstanceKeys()",
           "justification": "This method had a wrong name. It was only used internally as it provides no value for others."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.returnTypeChanged",
+          "old": "method long io.camunda.zeebe.protocol.record.value.EvaluatedDecisionValue::getDecisionVersion()",
+          "new": "method int io.camunda.zeebe.protocol.record.value.EvaluatedDecisionValue::getDecisionVersion()",
+          "justification": "This method returned the wrong type. It was only used internally as it provides no value for others."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "method io.camunda.zeebe.protocol.record.value.ImmutableEvaluatedDecisionValue.Builder io.camunda.zeebe.protocol.record.value.ImmutableEvaluatedDecisionValue.Builder::withDecisionVersion(long)",
+          "justification": "This method provided an input argument of the wrong type. It was only used internally as it provides no value for others."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.returnTypeChanged",
+          "old": "method long io.camunda.zeebe.protocol.record.value.ImmutableEvaluatedDecisionValue::getDecisionVersion()",
+          "new": "method int io.camunda.zeebe.protocol.record.value.ImmutableEvaluatedDecisionValue::getDecisionVersion()",
+          "justification": "This method returned the wrong type. It was only used internally as it provides no value for others."
```

Comment:
>It was only used internally as it provides no value for others

This is not entirely true. The Immutable record classes are exposed in our bom, for other JVM projects to use and depend on. It is possible that others are using this method. However, the `version` is unlikely to surpass MAX_INT. Combined with the JVM's ability to automatically convert these types (in many cases), I don't expect any problems.

🔧 Let's adjust the justification to mention this